projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-bar/wpcom-admin-bar.scss

```diff
@@ -108,11 +108,6 @@
 				margin-right: 0 !important;
 				.ab-item {
 					width: 52px;
-					svg {
-						width: 36px;
-						height: 36px;
-						padding: 4px 8px;
-					}
```

Comment:
This code can now be found here: https://github.com/Automattic/wp-calypso/blob/dfcc8001000b2c8ebf0a0005e767c4a78da3df98/apps/help-center/help-center.scss#L60-L62